microcosm_flask/conventions/health.py

```diff
@@ -60,49 +63,64 @@ class Health:
     def __init__(self, graph, include_build_info=True):
         self.graph = graph
         self.name = graph.metadata.name
+        self.checks = dict()
+
         if include_build_info:
-            self.checks = dict(
+            self.optional_checks = dict(
```

Comment:
A bit more of a reach, but I figured if were going to say "/health is *only* essential checks", then these wouldn't make the cut.